openassessment/assessment/api/peer.py

```diff
@@ -436,6 +436,8 @@ def get_assessment_median_scores(submission_uuid):
         assessments = [item.assessment for item in items]
         scores = Assessment.scores_by_criterion(assessments)
         return Assessment.get_median_score_dict(scores)
+    except PeerWorkflow.DoesNotExist:
+        return {}
```

Comment:
This was never called in a situation where no peer workflow existed before, and yes it did crash once that situation became a possibility.